typed-racket-lib/typed-racket/base-env/base-types-extra.rkt

```diff
@@ -17,7 +17,7 @@
 (define-other-types
   -> ->* case-> U Rec All Opaque Vector
   Parameterof List List* Class Object Values Instance Refinement
-  pred Struct Struct-Type Prefab Top Bot)
+  pred Struct Struct-Type Prefab Top Bot Distinction)
```

Comment:
It's not; that's what this except-in is for: https://github.com/racket/typed-racket/pull/170/files#diff-be2db6e0db44e2e94aa8b096cd9b3850R21